tokenizer_test.go

```diff
@@ -121,6 +128,7 @@ func TestEncodeWithAndWithoutOptions(t *testing.T) {
 			wantAttentionMask:     []uint32{0x1, 0x1},
 			wantIDs:               []uint32{101, 102},
 			wantTokens:            []string{"[CLS]", "[SEP]"},
+			wantOffsets:           []tokenizers.Offset{{0x0, 0x0}, {0x0, 0x0}},
```

Comment:
i'd expect this slice to be empty when offsets are not requested

Reply:
I think this test requests all options, but there are tests below where the offset is not requested and the returned slice is indeed empty.